src/plugins/sync-data-attrs/client.js

```diff
@@ -0,0 +1,18 @@
+var excludedAttrs = [
+	'intlOverrides',
+	'langDefault',
+	'oslo',
+	'timezone'
+];
```

Comment:
It would be nice to manage all of these in this one plugin, but unfortunately some of these are configurable options and some clients don't follow the defaults. Thus, it would be challenging to pull these in without breaking existing clients.